### DISCUSSION TITLE
How to build native runtime

### DISCUSSION THREAD

**Discussion by partyalarm** [amerkoleci/Vortice.Vulkan#47] [General]
Hello.
I would like to include a linux-arm64 runtime to run Vortice.VulkanMemoryAllocator on android,
How do you build the original VMA source code?

**Comment by partyalarm** [comment]
Self-resolved. Repository found.
https://github.com/amerkoleci/native_libs

**Reply by amerkoleci** [reply]
Yep, feel free to contibute to the repo if you add android or other platform support.

Thanks!

**Reply by partyalarm** [reply]
I made a prototype on Mac.
I am not familiar with Windows, so I am looking up how to write cmake.
https://github.com/partyalarm/native_libs/tree/main/build
